### PR TITLE
Feature/apple watch ecg

### PR DESCRIPTION
**Pull request by claytoncasey01**
@ZuSe Had to create a new pull request for some reason. Conflicts should be fixed now.

### CODE REVIEW

**Comment by ZuSe** [comment]
looks good, thanks!

**Review by ZuSe — APPROVED** [review]

**Comment by claytoncasey01** [comment]
@ZuSe Had to resolve some conflicts, those are fixed up now and good to merge after a review.

**Comment by denkoul** [comment]
What is the status of this? I am a big fan of your package and I am waiting for this feature. Thanks

File: packages/health/lib/src/health_data_point.dart [hunk 1]
```diff
@@ -3,21 +3,24 @@ part of '../health.dart';
 /// A [HealthDataPoint] object corresponds to a data point captures from GoogleFit or Apple HealthKit
 class HealthDataPoint {
   num _value;
+  List _values;
```
Comment (ZuSe) [inline packages/health/lib/src/health_data_point.dart:6]: Can you type it?

**Review by ZuSe — CHANGES_REQUESTED** [review]
Use strong types wherever possible

File: packages/health/lib/src/health_data_point.dart (hunk 1 above)
Comment (claytoncasey01) [reply packages/health/lib/src/health_data_point.dart:6]: Yeah I will get that done ASAP

**Comment by Moseco** [comment]
Is this PR still being worked on?

@claytoncasey01

**Comment by claytoncasey01** [comment]
> Is this PR still being worked on?
> 
> @claytoncasey01

Sorry I haven't had any time to continue working on this. I did it for a specific need for our company and may not be useful to anyone else. So we work with Hospitals who have access to recreate ECG from the voltage data, this change only returns an array of Voltage from the leads and time. If this is useful I can try to get that finished but you will not get any events, just an array of voltages.

**Pull request closed without merging**